website/content/en/preview/contributing/working-group.md

```diff
@@ -1,13 +1,69 @@
 ---
```

Comment:
nit: We should rename this filepath to `community-meetings` and then make sure that we have a redirect built-in from the old path to the new in our redirects